content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,209 @@
+:guilabel:`Multi-Step Routes` checkbox under the :guilabel:`Warehouse` heading. By doing so, the
+:guilabel:`Storage Locations` setting is automatically enabled as well. Finally, click
+:guilabel:`Save`.
+
+.. image:: putaway/multi-step-routes.png
    :align: center
+   :alt: Check multi-step routes box
+
+Put away products
+=================
+
+Putaway rules are commonly used to move products to specific locations for easy retrieval. Thus,
+products are grouped in the same :guilabel:`Product Category` when a single putaway rule captures
+the group's storage needs.
 
-Setting up a Putaway Rule
-=========================
+To manage where specific products are routed for storage, navigate to :menuselection:`Inventory -->
+Configuration --> Putaway Rules`. Use the :guilabel:`Create` button to configure a new putaway rule
+on a :guilabel:`Product` and/or :guilabel:`Product Category` that the rule affects.
 
-In some cases, like for a retail shop storing vegetables and fruits, we have to store products in
-different locations to maintain product quality.
+In the same line, the :guilabel:`When product arrives in` location is where the putaway rule is
+triggered to create an operation to move the product to the :guilabel:`Store to` location. The
+:guilabel:`Store to` location must be sublocation of the first (ie. `WH/Stock/Fruits` is a specific,
+named location inside `WH/Stock` to make the products stored here easier to find).
 
-Let’s suppose there are one warehouse location *WH/Stock* and two sub-locations
-*WH/Stock/Vegetables* and *WH/Stock/Fruits*.
+.. example::
+   Ensure all apples are stored in the fruits section by filling field :guilabel:`Store to` with the
+   location `WH/Stock/Fruits` when the :guilabel:`Product`, `Apple` arrives in `WH/Stock`. Repeat
+   this for all products and hit :guilabel:`Save`.
 
-To manage those locations, we will create putaway rules. To do so, open the *Inventory* app and go
-to :menuselection:`Configuration --> Putaway Rules`. Then, click on create and configure your first
-rule indicating the main location the product will enter before being redirected to the right
-location.
+   .. image:: putaway/create-putaway-rules.png
+      :align: center
+      :alt: Create putaway rules for apples and carrots.
 
 .. note::
-   The putaway rules can be defined either per product or per product category.
+  It is also possible to create and manage putaway rules for a single product by going to the
+  product page and clicking the :guilabel:`Putaway Rules` smart button at the top of the page. If
+  the button isn't there at first glance, select the :guilabel:`More` button at the top right to
+  view additional configuration options.
+
+.. _inventory/putaway/internal-stock-moves:
+
+.. tip::
+   Check whether putaway rules work as intended and using the summary of internal product movements.
+   In the :guilabel:`Inventory` app, select :menuselection:`Reporting --> Product Moves` and enable
```

Comment:
I believe in our writers workshop meeting, we decided to use :menuselection: for the "Inventory" app in this case.
```suggestion
   In the :menuselection:`Inventory` app, select :menuselection:`Reporting --> Product Moves` and enable
```